qutip/core/cy/qobjevo.pyx

```diff
@@ -312,6 +312,9 @@ cdef class QobjEvo:
             if _args is not None:
                 kwargs.update(_args)
             return QobjEvo(self, args=kwargs)(t)
+        if self.isconstant:
+            # When the QobjEvo is constant, it is usually made of only one Qobj
```

Comment:
```suggestion
            # For constant QobjEvo's, we sum the contained Qobjs directly in order to
            # retain the cached values of attributes like .isherm when possible, rather than
            # calling _call(t) which may lose this cached information.
```